app/src/Topbar.re

```diff
@@ -1,6 +1,8 @@
+open Atom;
+
 module JoinNetwork = {
   [@react.component]
-  let make = () => <button> {React.string("Join Network")} </button>;
+  let make = () => <Button style=Button.Styles.primary text="Join Network" />;
```

Comment:
Instead of having a text prop could we pass a children? Or could that potentially break the button styles/implementation?

Reply:
You're right, I think that's the proper react way to do it.